Makefile

```diff
@@ -103,10 +106,10 @@ start-android-emulator:
 
 .PHONY: prepare-android-emulator
 prepare-android-emulator:
-	@adb root
-	@adb shell avbctl disable-verification
-	@adb reboot
-	@sleep 60
+#	@adb root
```

Comment:
Commented out lines? If you intended to leave them in there, could you add an explanation?

Reply:
Forgot to delete them